website/content/en/docs/kubebuilder/references/client.md

```diff
@@ -150,28 +142,28 @@ Example:
 
 ```Go
 import (
-	"context"
-	"fmt"
-	"k8s.io/api/core/v1"
-	"sigs.k8s.io/controller-runtime/pkg/client"
-	"sigs.k8s.io/controller-runtime/pkg/reconcile"
+    "context"
+    "fmt"
+    "k8s.io/api/core/v1"
+    "sigs.k8s.io/controller-runtime/pkg/client"
+    ctrl "sigs.k8s.io/controller-runtime"
 )
 
-func (r *ReconcileApp) Reconcile(request reconcile.Request) (reconcile.Result, error) {
-	...
-
-	// Return all pods in the request namespace with a label of `app=<name>`
-	// and phase `Running`.
-	podList := &v1.PodList{}
-	opts := []client.ListOption{
-		client.InNamespace(request.NamespacedName.Namespace),
-		client.MatchingLabels{"app": request.NamespacedName.Name},
-		client.MatchingFields{"status.phase": "Running"},
-	}
-	ctx := context.TODO()
-	err := r.List(ctx, podList, opts...)
-
-	...
+func (r *ReconcileApp) Reconcile(req ctrl.Request) (ctrl.Request, error) {
```

Comment:
The Reconciler struct name in the new scaffolds are of the format `<kind>Reconciler`.
Please double check and update all Reconcile struct and method receiver names in this doc accordingly.
https://github.com/operator-framework/operator-sdk-samples/blob/master/go/kubebuilder/memcached-operator/controllers/memcached_controller.go#L46
```suggestion
func (r *AppReconciler) Reconcile(req ctrl.Request) (ctrl.Request, error) {
```